### ISSUE TITLE
Wrong window focused when Toastify gives back focus to Visual Studio

### ISSUE THREAD

**Issue by bolddp** [nachmore/toastify#6]
I'm not sure if this is a Toastify problem, it can also be related to how Visual Studio handles returned focus or even how I've configured my window layout.

I have the code on my left monitor in the main window, and then I have Solution Explorer, Properties etc. on the right monitor in free-flowing windows.

To repro for me:
- Put cursor in code editor in main window
- Press Ctrl+Alt+UpArrow to halt Spotify playback.
- When the toast has faded out (not when fading in), focus is not returned to the main window but instead to the free-flowing window that was last focused before the main window got focus.

This causes Visual Studio to move focus to the Solution Explorer on every track change, potentially deleting files if I was in the process of deleting characters in the code... :)

**Comment by nachmore** [comment]
Thanks for the bug - this is indeed a weird one. I try not to futz around with focus as much as possible to avoid these kinds of bugs, so this is an interesting one. 

For me VS has Ctrl+Alt+Up assigned to "InteractiveConsole.SearchHistoryPrevious" which, depending on what I hit next can jump to random windows within VS - could it be that you have a conflicting VS keybinding that is causing this to happen?

**Comment by bolddp** [comment]
I don't think that is the problem, since the same thing happens when the toast is displayed due to a normal track change.

Another observation, that may point toward it being a VS issue: The VS window that gets focus instead of the main window is always the free-floating window that was last focused before the main window was focused.

If I find the time I'll clone the project and see if I can investigate a little on my own as well.

**Comment by bolddp** [comment]
I downloaded the project now and as far as I can see I'll have to live with this :) I made an attempt to capture the foreground window in the Toast.FadeIn method and then set it to foreground in FadeOut, but it made no difference.

And when trying to query for the active MDI child I didn't get any window handle. I suppose VS isn't an MDI application since it's an application type that seem to be deprecated nowadays.

Thanks for an awesome application anyway!
